elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -163,6 +166,10 @@ def render(element_html, data):
             html = 'ERROR: No true answer'
         else:
             html = '(%s) %s' % (correct_answer['key'], correct_answer['html'])
```

Comment:
We actually are just creating the "correct answer" panel straight in the python file without any mustache templates (and html parameters).  Would you mind refactoring this so that it uses the `pl-multiple-choice.mustache` file, like the checkbox elem?

Reply:
done in a2cd19e6b1a46ce12794ffbdc8d89da023e7aab2